lib/ApiRequestor.php

```diff
@@ -145,16 +145,26 @@ private static function _defaultHeaders($apiKey)
 
         $langVersion = phpversion();
         $uname = php_uname();
-        $curlVersion = curl_version();
+
+        $httplib = 'curl not installed';
```

Comment:
so `curl uknown` or just `unknown`? I was assuming that if we parsed the header we'd expect to see a `curl prefix`.